server/src/main/java/com/flipkart/varadhi/config/ServerConfiguration.java

```diff
@@ -28,6 +28,7 @@ public class ServerConfiguration {
 
     private AuthorizationOptions authorization;
 
+    private boolean authorizationServerEnabled;
```

Comment:
can we make it implicit.
Can server be disabled, if I configure DefaultAuthorizationProvider as the authzProvider?

Reply:
server goes in with the default provider, without that there is no way to set the role assignments

Reply:
so we will have to do the reverse, if default auth provider is enabled, server is enabled by default

Reply:
done